spydrnet_physical/util/ConnectPoint.py

```diff
@@ -171,13 +177,21 @@ def __str__(self) -> str:
         return "%5d %5d %5d %5d [%s]" % (self.from_x, self.from_y, self.to_x, self.to_y, self._level)
 
     def __mul__(self, scale):
+        '''Returns to and from coordinates after multipluing them with the scaling factor
+        '''
+        # But why not changing the original connectpoint
```

Comment:
It is because, when you perform `new_pt = pt * 2` you don't want pt to change, just a return value should be different